nodeLib/buildNodePackage.nix

```diff
@@ -536,29 +536,43 @@ let
         export NODE_PATH=$NODE_MODULES:$NODE_PATH
         # Check if the current directory contains the package.json for
         # this package.
-        if python -c "import json; assert json.load(open('package.json'))['name'] == '$fullName'" 2>/dev/null; then
+        py_cmd='import json; print(json.load(open("package.json"))["name"])'
+        if [[ -e package.json ]] && \
+            [[ $(python -c "$py_cmd" 2>/dev/null) == "$fullName" ]]; then
```

Comment:
I can't tell where $fullName gets defined, but it must work because it was in the script before.